it/v1/endpoints/AmendPensionsReliefsControllerISpec.scala

```diff
@@ -241,11 +268,14 @@ class AmendPensionsReliefsControllerISpec extends IntegrationBaseSpec {
           (BAD_REQUEST, "INVALID_TAXABLE_ENTITY_ID", BAD_REQUEST, NinoFormatError),
           (BAD_REQUEST, "INVALID_TAX_YEAR", BAD_REQUEST, TaxYearFormatError),
           (BAD_REQUEST, "INVALID_PAYLOAD", INTERNAL_SERVER_ERROR, InternalError),
+          (BAD_REQUEST, "INVALID_CORRELATIONID", INTERNAL_SERVER_ERROR, InternalError),
```

Comment:
I think we should also split this up to have two sequence variables (errors + extraTysErrors) and concatenate them together like we did for the service.